pkg/collector/corechecks/embed/jmx/check.go

```diff
@@ -82,6 +85,10 @@ func (c *JMXCheck) ID() check.ID {
 	return c.id
 }
 
+func (c *JMXCheck) IsTelemetryEnabled() bool {
```

Comment:
We might want to hardcode this to false. I don't think any metric is "routed" through this check. This thing is just some kind of hack to have a long running jmxfetch.

Reply:
I don't support it right now, but if later we want to have for instance the execution time of each run of the JMX check, I think we would be able to enable/disable it through this flag because it'll pass by the JMXCheck.GetMetricStats(), WDYT?